pyproject.toml

```diff
@@ -103,7 +103,7 @@ core = [
 
 	# for distutils
 	"jaraco.collections",
-	"jaraco.functools",
+	"jaraco.functools>=4",
```

Comment:
I guess this should be changed here too, since that repo is where the vendored `distutils` (and hence, these deps) is coming from: https://github.com/pypa/distutils/blob/378984e02edae91d5f49425da8436f8dd9152b8a/pyproject.toml#L23

Reply:
Isn't that what this changes? 🤔

Reply:
Normally, any change done in the `_distutils` folder should be done in the distutils repo *instead*.

I'm just not sure if pyproject dependencies follow the same rule (ie: are they refreshed from distutils or are they manually updated). So you could do the same change over there.

Anyway, maintainers will let you if that's the case.

Reply:
Ahh makes sense.

I have created a pull request in the `distutils` repo 👍 
https://github.com/pypa/distutils/pull/301